fineract-provider/src/main/java/org/apache/fineract/portfolio/loanproduct/api/LoanProductsApiResourceSwagger.java

```diff
@@ -1062,7 +1062,7 @@ static final class GetLoanProductsRepaymentStartDateType {
             private GetLoanProductsRepaymentStartDateType() {}
 
             @Schema(example = "1")
-            public Long id;
+            public Integer id;
```

Comment:
This looks wrong to me based on EnumOptionData id field type is Long

Reply:
Done.